src/usr/local/www/system_advanced_admin.php

```diff
@@ -597,4 +597,4 @@
 	log_error(gettext("webConfigurator configuration has changed. Restarting webConfigurator."));
 	send_event("service restart webgui");
 }
-?>
\ No newline at end of file
+?>
```

Comment:
I tried a 2nd time - the GitHub online editor is doing that to me. But if I get the version before and after the change I can't see any difference.